### PR TITLE
Account for the openshift-state-metrics component

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

**Comment by rh-max** [comment]
@bmcelvee Could you PTAL and merge + cherrypick to `enterprise` `4.2`, `4.3`, and `4.4`? Thank you.

**Comment by rh-max** [comment]
@bmcelvee The BZ is https://bugzilla.redhat.com/show_bug.cgi?id=1813211.

**Comment by rh-max** [comment]
@bmcelvee This received a technical review, and no further (QE) review needed.

File: modules/monitoring-stack-components-and-monitored-targets.adoc [hunk 1]
```diff
@@ -31,6 +31,9 @@ The monitoring stack includes these components:
 |`kube-state-metrics`
 |The `kube-state-metrics` exporter agent converts Kubernetes objects to metrics that Prometheus can use.
 
+|`openshift-state-metrics`
+|The `openshift-state-metrics` exporter expands upon `kube-state-metrics` by adding metrics for OpenShift-specific resources.
```
Comment (bmcelvee) [inline modules/monitoring-stack-components-and-monitored-targets.adoc:35]: Just one thing: 
s/OpenShift-specific/{product-title}-specific

**Comment by bmcelvee** [comment]
Just one change request, then I'll merge and cherry-pick

**Comment by rh-max** [comment]
@bmcelvee It's there, thank you.

**Pull request merged**